src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -185,4 +185,18 @@ export default class PaypalCommercePaymentProcessor {
         return updatedStyle;
     };
 
+    private _processNotEligible(buttonParams: ButtonsOptions, fundingKey?: keyof PaypalCommerceSDKFunding): void {
+        if (fundingKey === 'PAYLATER') {
+            buttonParams.fundingSource = this._paypal?.FUNDING.CREDIT;
```

Comment:
Should not we use `PAYLATER ` source here?

Reply:
Should. You are right. Thank you!